src/parsing/parsing.js

```diff
@@ -128,7 +130,10 @@
       }
     }
     var parsed = parser(string.substr(0, n));
-    return parsed == undefined ? undefined : [parsed, string.substr(n)];
```

Comment:
I thought about it, but other places seem to do what I have done rather than use a ternary.